libs/auth/src/lib/services/auth.service.ts

```diff
@@ -77,6 +77,13 @@ export class AuthService {
     );
   }
 
+  getAnonymousUserProfile(): ORD.ObservableRemoteData<ApiError, User> {
```

Comment:
Can we completely remove the user profile from the client side?

Reply:
yes, it's removed now